packages/xo-web/src/common/intl/messages.js

```diff
@@ -1534,6 +1534,7 @@ const messages = {
   unknownOsName: 'Unknown',
   unknownOsKernel: 'Unknown',
   unknownOriginalTemplate: 'Unknown',
+  unavailablebleOn: 'Unstable feature, insecure for the host, usage is discouraged',
```

Comment:
What do you think of: `unstableHostFeature`?
Else, don't forget to fix your typo

Reply:
Yes, this message key is not precise enough, what do you both think of `nestedVirtualizationWarning`?